app/celery/tasks.py

```diff
@@ -341,7 +348,15 @@ def save_smss(self, service_id: Optional[str], signed_notifications: List[Any],
             sms_queue.acknowledge(receipt)
             current_app.logger.info(f"Batch saving: {receipt} removed from buffer queue.")
         else:
-            put_batch_saving_bulk_processed(metrics_logger, 1)
+            if Config.FF_PRIORITY_LANES:
+                put_batch_saving_bulk_processed(
+                    metrics_logger,
+                    1,
+                    notification_type=SMS_TYPE,
+                    priority=notification_priority,
+                )
+            else:
+                put_batch_saving_bulk_processed(metrics_logger, 1, "should not show up", "should not show up")
```

Comment:
error logging might work better here instead

Reply:
ok, made them optional, default `None` and if `put_batch_saving_bulk_processed()` doesn't want them to be `None` (ie when the FF is on) it logs a warning instead of setting the dimensions.